api/app/signals/apps/signals/migrations/0085_auto_20191216_1251.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.8 on 2019-12-16 11:51
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+copy_category_departments = """
+INSERT INTO signals_categorydepartment(is_responsible, can_view, category_id, department_id)
+SELECT True, True, category_id, department_id FROM signals_category_departments;
```

Comment:
Misschien kleine comment toevoegen waarom is_responsible en can_view in eerste instantie altijd true zijn.

Reply:
Comment toegevoegd